gemfiles/Gemfile-rails-4-2

```diff
@@ -6,3 +6,4 @@ gemspec path: ".."
 gem "rails", "~> 4.2.0"
 gem "sprockets", "~> 4.0.0.beta2"
 gem "sprockets-rails", "~> 3.0.4"
+gem 'nokogiri', '~> 1.6.8'
```

Comment:
Is there a way to version check this so that we use a later version on Rubies that nokogiri still supports?